README.md

```diff
@@ -1195,6 +1196,31 @@ datetime                           fromAddress  fromAddressInExchange
 2019-04-30 15:17:28+00:00  0x876eabf441b2ee...                   True  0x1f4a90043cf2d...                False  0xc85892b9ef8c64...   20544.42975
 ```
 
+### Get Metric
```

Comment:
having `Get Metric` in the documentation is an implementation detail that could be omitted

Reply:
Why wouldn't it be in the documentation?